### ISSUE TITLE
make provision does not decrypt

### ISSUE THREAD

**Issue by ivomarino** [ttssdev/appflow#13]
since `ansible 2.2.0.0` the `make provision` does not automatically decrypt tenant's configs, seems to be related to https://github.com/ansible/ansible/issues/18364.

**Comment by ivomarino** [comment]
a current workaround is to stick with `ansible 2.0.2.0`:

```
% brew unlink ansible
% homebrew/versions/ansible20
```

**Comment by ocean90** [comment]
Instead of unlinking you can also switch the version. Use `info` to check which versions are installed and then `switch` to a version before 2.2:

```
$ brew info ansible
ansible: stable 2.2.0.0 (bottled), HEAD
Automate deployment, configuration, and upgrading
https://www.ansible.com/
/usr/local/Cellar/ansible/2.1.2.0 (10,428 files, 104.7M)
  Poured from bottle on 2016-10-20 at 11:24:13
/usr/local/Cellar/ansible/2.2.0.0 (10,880 files, 110.1M)
  Poured from bottle on 2016-11-08 at 00:48:39
/usr/local/Cellar/ansible/2.2.0.0_1 (10,873 files, 110.1M) *
  Poured from bottle on 2016-11-08 at 18:20:11
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/ansible.rb
==> Dependencies
Build: pkg-config ✔
Required: libyaml ✔, openssl@1.1 ✔
==> Requirements
Required: python ✔
$ brew switch ansible 2.1.2.0
```

Or build your own version using the devel branch: `brew reinstall ansible --HEAD`

**Comment by ivomarino** [comment]
great hint, thanks. It told me before that two version of the same app can't be installed on the same instance, `brew` suggested me to `unlink`.

**Comment by ivomarino** [comment]
current one-liner: `brew tap homebrew/versions ; brew unlink ansible ; brew install ansible20` -- @ocean90 maybe we can integrate `switch` and `ansible21` also here.

**Comment by swissspidy** [comment]
Fix in the stable 2.2 branch: https://github.com/ansible/ansible/commit/366bfe14c39d6aee1604928edb2c76589750a516

**Comment by ivomarino** [comment]
@swissspidy if seen this, yes. Does not seem to be true anyway, for some reason. `2.2` makes still stress here (tested also on @grappler's Mac).

**Comment by ivomarino** [comment]
this brings ansible to 2.3.0 which is not affected by decrypt issue:
`brew unlink ansible ; brew unlink ansible20 ; brew reinstall ansible --HEAD`

**Comment by 89luca89** [comment]
As there are 2 type of bug, required version is 2.0.x:
`2.3.x` works for the encrypt/decrypt bug, but ansible versions `2.2+` have a bug that some checks are randomly skipped (there are already bug reports open)
I hope they will fix for 2.3-stable

For now use `stable-2.0`
If using `pip` just use:
`sudo pip install git+git://github.com/ansible/ansible.git@stable-2.0`

**Comment by ivomarino** [comment]
close due to ansible bug, solution is posted here anyway.
